service/controller/rbac/resource/fluxauth/create.go

```diff
@@ -244,13 +247,6 @@ func (r *Resource) createOrUpdateClusterRoleBinding(ctx context.Context, ns core
 		return microerror.Mask(err)
 	} else if needsUpdateClusterRoleBinding(clusterRoleBinding, existingClusterRoleBinding) {
 		r.logger.LogCtx(ctx, "level", "info", "message", fmt.Sprintf("updating cluster role binding %#q for Automation SA in namespace %s", clusterRoleBinding.Name, ns.Name))
-		clusterRoleBinding.Subjects = append(existingClusterRoleBinding.Subjects,
```

Comment:
We do not append the subject to the existing clusterrolebinding to avoid appending it multiple times and making the resource larger and larger.